lib/mpl_toolkits/axes_grid1/axes_size.py

```diff
@@ -17,14 +21,41 @@ class (or others) to determine the size of each Axes. The unit
 
 class _Base:
     def __rmul__(self, other):
+        if not isinstance(other, Real):
+            return NotImplemented
+        return Fraction(other, self)
+
+    def __mul__(self, other):
+        if not isinstance(other, Real):
+            return NotImplemented
         return Fraction(other, self)
 
+    def __div__(self, other):
+        if not isinstance(other, Real):
+            return NotImplemented
+        return Fraction(1/other, self)
+
     def __add__(self, other):
         if isinstance(other, _Base):
             return Add(self, other)
         else:
             return Add(self, Fixed(other))
 
+    def __neg__(self):
+        return -1 * self
+
+    def __radd__(self, other):
+        # other cannot be a _Base instance, because A + B would trigger
+        # A.__add__(B) first.
+        return Add(self, Fixed(other))
+
+    def __sub__(self, other):
+        if isinstance(other, _Base):
```

Comment:
I think `__sub__` can just be defined (in all cases, including the exception-raising one) as `self + (-other)`?